### ISSUE TITLE
Config Parameter: Disallow users to change any personal information

### ISSUE THREAD

**Issue by jonathanmmm** [nextcloud/server#30396]
Hi,

I know there is already a feature to disable chaning display name (which right now also disables changing the email): https://github.com/nextcloud/server/issues/16823.

I would like a feature to disable every thing, as users can change their profile stuff via another backend (and nextcloud is synching username, etc. via ldap, sadly right now, not phone number, etc). So only readonly and everything from ldap.
Users could get confused if they change it in nextcloud and it is not reflected in other services.

This issue is also related:
https://github.com/nextcloud/server/issues/11806 or here https://github.com/nextcloud/server/issues/16823

**Comment by sevmonster** [comment]
This doesn't make a lot of sense until more apps support updating user attributes. A push should be made to get out both at once, in my mind.
https://github.com/nextcloud/user_saml/issues/585

**Comment by jonathanmmm** [comment]
> This doesn't make a lot of sense until more apps support updating user attributes. A push should be made to get out both at once, in my mind. [nextcloud/user_saml#585](https://github.com/nextcloud/user_saml/issues/585)

True, right now (not from saml and not from LDAP), the other information is not carried over, so people could get confused, changing there information in Nextcloud but it is not reflected in LDAP/other services (I use LDAP, but read only, users have therefore a way to alter there information, but outside of nextcloud).
